primitives/runtime-interface/test/src/lib.rs

```diff
@@ -176,62 +176,85 @@ fn test_versionining_register_only() {
 	call_wasm_method::<HostFunctions>(wasm_binary_unwrap(), "test_versionning_register_only_works");
 }
 
+fn run_test_in_another_process(
+	test_name: &str,
+	test_body: impl FnOnce(),
+) -> Option<std::process::Output> {
+	if std::env::var("RUN_FORKED_TEST").is_ok() {
+		test_body();
+		None
+	} else {
+		let output = std::process::Command::new(std::env::current_exe().unwrap())
+			.arg(test_name)
+			.env("RUN_FORKED_TEST", "1")
+			.output()
+			.unwrap();
+
+		assert!(output.status.success());
+		Some(output)
+	}
+}
+
 #[test]
 fn test_tracing() {
```

Comment:
This one was flaky?

Reply:
` while cargo test --release -p sp-runtime-interface-test; do true; done`